core/sdk/src/utils/logger.rs

```diff
@@ -20,7 +20,7 @@ use std::{str::FromStr, sync::atomic::AtomicU32};
 pub static LOG_COUNTER: AtomicU32 = AtomicU32::new(1);
 
 pub fn init_logger(name: String) -> Result<(), fern::InitError> {
-    let log_string: String = std::env::var("RUST_LOG").unwrap_or_else(|_| "debug".into());
```

Comment:
Do we want to default to `trace`? I thought the plan was to still default to `debug` but in situations where we wanted to display `log_signal` output we'd just set the log level to `trace`?

Reply:
'll change it to debug, missed it

Reply:
@brendanobra There is only one ripple.service file for both prod and VBN, so do we want to default to INFO now?

Reply:
Why do we need to change it from what it was ("debug") at all? Wasn't VBN/PROD builds working fine as it was? And this isn't needed for the changes to log_signal is it?